core/src/main/java/com/linecorp/armeria/server/PeriodicUnloggedExceptionsReporter.java

```diff
@@ -31,11 +31,11 @@
 import io.micrometer.core.instrument.Counter;
 import io.micrometer.core.instrument.MeterRegistry;
 
-final class DefaultUnhandledExceptionsReporter implements UnhandledExceptionsReporter {
+final class PeriodicUnloggedExceptionsReporter implements UnloggedExceptionsReporter {
```

Comment:
To be clear, the original intention of suggesting `PeriodicExceptionsReporter` was to intentionally keep the type of reported exceptions vague in case we accept a condition on which exceptions are reported.

I understood that we won't be changing the behavior of the reporter after this iteration, so I think it's fine to just name as `DefaultUnloggedExceptionsReporter` (It's internal anyways)

Reply:
Then, let me revert the commit. 😉